src/components/tabsv2/TabsV2.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { FunctionComponent } from "react";
+import styled, { css } from "styled-components";
+import theme from "../../theme";
+import { Text } from "../text";
+import {
+  ButtonNextProps,
+  ButtonPreviousProps,
+  TabProps,
+  TabsContent,
+  TabsProps,
+  Step,
+  TabsWrapper,
+  TabsList,
+  NextButton,
+  PreviousButton,
+  TabsCover,
+  TabsStatusButtons,
+} from "../tabs";
+
+export const TabV2Container: any = styled.div<TabProps>`
+  padding: ${theme.spacing.spacing04};
+  background-color: ${theme.color.background.ui03};
+  border-right: 1px solid ${theme.color.border.border01};
+  border-bottom: 1px solid transparent;
+  width: 17rem;
+  &:last-child {
+    border-right: none;
+    &.tab-active {
+      border-right: 1px solid ${theme.color.border.border01};
+    }
+  }
+
+  &:hover {
+    cursor: ${({ disabled }) => (disabled ? `auto` : `pointer`)};
+    background-color: ${({ disabled }) =>
+      disabled
+        ? `${theme.color.background.ui03}`
+        : `${theme.color.background.ui01}`};
+  }
+
+  ${({ wizard }) =>
+    wizard &&
+    css`
+      width: auto;
+      flex: 1;
+      &:last-child {
+        border-right: none;
+        &.tab-active {
+          border-right: none;
+        }
+      }
+    `}
+  &:first-child {
+    border-top-left-radius: ${theme.radiusDefault};
+  }
+  &.tab-active {
+    background-color: ${theme.color.background.ui01};
+    border-bottom: 1px solid ${theme.color.background.ui01};
+    z-index: ${theme.zindex.default};
+  }
+  &.tab,
+  &.tab-active {
+    &:first-child {
+      & > div:first-child {
+        background-color: ${({ wizard }) =>
+          wizard ? theme.color.background.ui05 : ""};
```

Comment:
This whole styled component should be re-used. `TabsV2Container` should extend from the first version and add only this piece on top.